dom/META.yml

```diff
@@ -2,13 +2,21 @@ links:
 - product: firefox
   results:
   - subtest: 'Document interface: attribute origin'
-    test: interfaces.html
```

Comment:
FYI @jgraham - seems that interfaces doesn't actually resolve to a test name (in-so-far as wpt.fyi treats it as a test name). I guess technically `interfaces.html` is a source file that explodes into 2 variants.